.github/workflows/mac.yml

```
@@ -11,15 +11,15 @@ jobs:
   test:
     name: Compile and test planner
     timeout-minutes: 60
-    runs-on: macos-10.15
+    runs-on: macos-11
```

```suggestion
    runs-on: [macos-11, macos-12]
```
Or use macos-12 instead of macos-11?

---

I guess there is a reason why macos-latest is still pointing to macos-11. Could it be the case that it is still the more common version? And I am not convinced that we have sufficient benefit of running two versions with the increased environmental footprint.